github-dark.css

```diff
@@ -1196,20 +1196,30 @@
     .pull-request-tab-content .diff-view .file-header {
       position: sticky !important;
       position: -webkit-sticky !important;
-      top: 40px !important;
+      top: 39px !important;
       z-index: 6 !important;
     }
-    .pull-request-tab-content .file-header {
-      padding: 1px 10px !important;
+    [id^=diff-].selected-line {
+      position: relative !important;
+      top: -120px !important;
+      left: -10000px !important;
     }
-    .pull-request-tab-content .diff-view .file-actions,
-    .pull-request-tab-content .diff-view .file-info {
-      margin-top: -1px !important;
+    [id^=diff-].selected-line:before {
+      content: attr(data-line-number) !important;
+      position: absolute !important;
+      top: 120px !important;
+      left: 10000px !important;
+      background-color: inherit !important;
+      color: inherit !important;
+      box-sizing: border-box !important;
+      display: block !important;
+      padding: .04em 10px !important;
```

Comment:
I think you can do without specifying vertical padding, just specify horizontal only.

Reply:
for me `.04em` is OK since `.03em` then I see a black underline

![capture](https://user-images.githubusercontent.com/31389848/44166111-7f9e9900-a0c2-11e8-98e2-8a18dd212e33.PNG)

So idk.

Reply:
I didn't mean 0, but specifying them individually so the original value is unchanged:

````css
padding-left: 10px !important;
padding-right: 10px !important;
````
But anyway, I will retest a better solution later.

Reply:
[test comment, please ignore]

```suggestion
      padding-left: 10px !important;
      padding-right: 10px !important;
```